abfe_tutorial/python_tutorial.ipynb

```diff
@@ -0,0 +1,1190 @@
+{
```

Comment:
<p>"which describes all the particles in the simulation"</p><p> </p><p>I know this comes from somewhere else but this might be a bit confusing, how about "to define the entire system"?</p><p> </p><p>Also protocol -&gt; <code>Protocol</code>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenFreeEnergy/ExampleNotebooks/pull/203/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>